airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -1390,6 +1379,12 @@ def download_file(
             Default: True.
         :return: the file name.
         """
+        self.log.info(
+            "This function shadows the 'download_file' method of S3 API, but it is not the same. If you "
+            "want to use the original method from S3 API, please call "
+            "'S3Hook.get_conn().download_file()'"
+        )
+
```

Comment:
```suggestion
        self.log.info(
            "This function shadows the 'download_file' method of S3 API, but it is not the same. If you "
            "want to use the original method from S3 API, please call "
            "'S3Hook.get_conn().download_file()'"
        )

```

https://github.com/apache/airflow/pull/37211

Reply:
The suggestion is the same as the original?

Reply:
reverted the change.